llvm/lib/CodeGen/TargetPassConfig.cpp

```diff
@@ -1537,6 +1539,12 @@ void TargetPassConfig::addBlockPlacement() {
   }
 }
 
+void TargetPassConfig::addBlockShuffling() {
+  if (addPass(&MachineBlockShufflingID)) {
+  
+  }
+}
+
```

Comment:
I guess this is unneeded

Reply:
the whole pass config (looks like) is a part of the old pass manager, but I hope it could be triggered by a special way